### PR TITLE
Fix issue with Integer Division

### PR DESCRIPTION
**Pull request by jasondaming**
Credit [here](https://www.chiefdelphi.com/t/falcon-fx-velocity-pid-and-stubborn-error-from-getclosedlooperror/392357/6?u=jdaming) for discovering the bug

@AustinShalit / @PeterJohnson this should probably be pushed ASAP.  I have not tested it.

### CODE REVIEW

**Comment by prateekma** [comment]
Closes #190

File: frc_characterization/logger_analyzer/templates/Robot.java.mako [hunk 1]
```diff
@@ -152,10 +152,10 @@ public class Robot extends TimedRobot {
           % if not brushed:
       CANEncoder encoder = motor.getEncoder();
           % else:
-      CANEncoder encoder = motor.getEncoder(EncoderType.kQuadrature, ENCODER_EDGES_PER_REV);
+      CANEncoder encoder = motor.getEncoder(EncoderType.kQuadrature, ENCODER_EPR);
```
Comment (sciencewhiz) [inline frc_characterization/logger_analyzer/templates/Robot.java.mako:155]: Why change the name? It won't use the constant, right?

Comment (jasondaming) [reply frc_characterization/logger_analyzer/templates/Robot.java.mako:155]: ENCODER_EPR is just the integer type version of ENCODER_EDGES_PER_REV.  We could use the same variable and cast to type or something, honestly this seemed simplest

**Comment by calcmogul** [comment]
Have all the review comments been resolved?

**Review by calcmogul — APPROVED** [review]

**Pull request merged**